content/tooling/integrations/github.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "GitHub Actions"
+new: true
+---
+
+# Deploy your Workers application with GitHub Actions
+
+If you'd like to automate deploying your Workers application, we've published a GitHub Action to simplify the process. GitHub Actions allows you to build continuous integration workflows for your GitHub projects, and with [`wrangler-action`](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler), you can customize your workflow to deploy your project in situations like:
+
+- When a new commit is merged to your `master` branch
+- On a `cron`-like schedule, such as every hour, or daily
+- When an API "dispatches" an event to your GitHub repo
+
+First, we'll explore the basics of using the action, and explain how to configure your project to use it. After the basics, we'll cover each of the above deployment situations in detail, with a brief explanation and code example to help you get started. If you'd like to see the action itself, you can [find it in GitHub's marketplace](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler).
+
+## Usage
+
+Add `wrangler-action` to the workflow for your Workers application. If you haven't set up a GitHub Actions workflow for your repo yet, you can create a new file called `.github/workflows/deploy.yml`, and add the below workflow, to start:
+
+```yaml
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish
+        uses: signalnerve/wrangler-action@0.1.4
+        with:
+          apiKey: ${{ secrets.apiKey }}
+          email: ${{ secrets.email }}
+```
+
+## Configuration
+
+You'll need to configure Wrangler using GitHub's Secrets feature - go to "Settings -> Secrets" and add your Cloudflare API key and email (for help finding these, see the [Workers documentation](https://developers.cloudflare.com/workers/quickstart/#finding-your-cloudflare-api-keys)). Your API key and email are encrypted by GitHub, and the action won't print them into logs, so they should be safe!
```

Comment:
We don't want to reiterate this information or maintain the Github actions instructions, so I suggest we just point to github's article and the finding api key article
```suggestion
You'll need to configure the API keys for Wrangler using GitHub's Secrets feature. First, [grab your Cloudflare API Key](https://developers.cloudflare.com/workers/quickstart/#finding-your-cloudflare-api-keys) then [add it to Github Secrets](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables). Your API key and email are encrypted by GitHub and the action won't print them into logs, so they should be safe!
```